src/Widgets/WindowSwitcher.vala

```diff
@@ -95,27 +133,18 @@ namespace Gala {
                 (uint8) (rgba.alpha * 255)
             );
 
-            if (granite_settings.prefers_color_scheme == Granite.Settings.ColorScheme.DARK) {
-                wrapper_background_color = "#333333";
-                caption_color = "#fafafa";
-            }
-
-            back_color = Clutter.Color.from_string (wrapper_background_color);
-            rect_radius = WRAPPER_BORDER_RADIUS * scaling_factor;
-
-            var layout = new Clutter.FlowLayout (Clutter.FlowOrientation.HORIZONTAL);
-            container = new Clutter.Actor ();
-            container.layout_manager = layout;
-            container.reactive = true;
-            container.button_press_event.connect (container_mouse_press);
-            container.motion_event.connect (container_motion_event);
-
             indicator = new RoundedActor (accent_color, WRAPPER_BORDER_RADIUS * scaling_factor);
 
             indicator.margin_left = indicator.margin_top =
                 indicator.margin_right = indicator.margin_bottom = 0;
             indicator.set_pivot_point (0.5f, 0.5f);
 
+            var caption_color = "#2e2e31";
+
+            if (granite_settings.prefers_color_scheme == Granite.Settings.ColorScheme.DARK) {
+                caption_color = "#fafafa";
+            }
+
```

Comment:
@danrabbit was there any way to get this from the style context, too?

Reply:
I keep getting a segfault trying to grab it, so I figured it was better to leave for another branch